cli/popper/utils.py

```diff
@@ -3,7 +3,8 @@
 import sys
 import time
 import yaml
-
+import git
```

Comment:
I believe we don't need this here anymore

Reply:
yup , removing